### PR TITLE
core/radvd Slightly Reorganise UI for disabling of DNS advertisements

### PR DESCRIPTION
**Pull request by pmhausen**
https://github.com/opnsense/core/issues/5919

### CODE REVIEW

File: src/www/services_router_advertisements.php [hunk 1]
```diff
@@ -420,10 +420,7 @@ function addRow() {
                       </div>
                       <br />
                       <input id="rasamednsasdhcp6" name="rasamednsasdhcp6" type="checkbox" value="yes" <?=!empty($pconfig['rasamednsasdhcp6']) ? "checked='checked'" : "";?> />
```
Comment (fichtner) [inline src/www/services_router_advertisements.php:422]: I think we should move both checkboxes into a new row with "DNS options" ?

Their help text is correct after b504bc8 (dns settings includes search list and servers

**Comment by fichtner** [comment]
mind if I take this over?

**Comment by fichtner** [comment]
@pmhausen thanks, if you can please have a brief look at https://github.com/opnsense/core/commit/68894e989dc890954746507fe85cd4f5c087d94a to see if it makes sense (help texts)

Ok, I'll try to place the plugin issue in the appropriate queue.

**Comment by pmhausen** [comment]
I find the wording "If a setting here is used the information below is ignored" a bit clumsy.

How about "If checked no DNS information is sent and the next setting concerning DHCPv6 is ignored."

**Comment by fichtner** [comment]
I tend to agree. The text, however, is trying to explain both settings at the same time. The below configuration is not DHCPv6 -- only the checkbox option above is.

**Comment by fichtner** [comment]
@pmhausen how about 1c322498c90 ? might be a little over the top but now the checkboxes do what the configuration actually does (disable other checkbox, hide custom dns options)

**Pull request closed without merging**